src/modules/layout/components/Header.js

```diff
@@ -108,17 +109,13 @@ class Header extends React.PureComponent<PropsType> {
     this.props.navigation.navigate('Settings')
   }
 
-  isPeeking (): boolean {
-    return this.props.peeking
-  }
-
   onShare = async () => {
     const { navigation, t } = this.props
     const sharePath: ?string = navigation.getParam('sharePath')
     if (!sharePath) {
-      return console.error('sharePath is undefined')
+      console.error('sharePath is undefined')
```

Comment:
I guess, we should only show the `Share` Menu-option, if there's a specific `sharePath` given in the navigation params. (a general integreat link doesn't make sense here imo).

Reply:
Why not? Just recommending integreat isn't that bad, is it?

Reply:
Reverted